Sources/Paywalls/PaywallData.swift

```diff
@@ -15,6 +15,37 @@
 
 import Foundation
 
+/// Represents countries where currencies typically have zero decimal places
+public struct ZeroDecimalPlaceCountries: Codable, Sendable, Hashable, Equatable {
+
+    /// Storefront IDs that should typically display zero decimal places
+    public var apple: Set<String> {
+        _apple
+    }
+
+    private var google: Set<String> {
```

Comment:
Do we need to parse Google's countries at all? We can probably just ignore the key in the response I think? (Or even not send it from the backend when using an iOS API key if possible)

Reply:
Done!